src/work/Work.h

```diff
@@ -97,6 +97,16 @@ class Work : public WorkParent
     void advance();
     void reset();
 
+    size_t getMaxRetries()
```

Comment:
I don't think those accessors add much value: you're just testing that the objects got initialized with the right parameters